### DISCUSSION TITLE
QTable Custom Filter, filter According to a raw value

### DISCUSSION THREAD

**Discussion by mecjos** [quasarframework/quasar#12030] [General - Components / Directives / etc]
I want to filter table raws according to their activation status. I couldn't find how I will properly use filter-method to do that. I will not give an input for filtering. That method will must filter rows according to situation whether status is active or not?

**Comment by MilosPaunovic** [comment]
Create a computed property for table rows and do the filtering you need there:
[https://codepen.io/milospaunovic/pen/rNGoqWB?editors=1011](https://codepen.io/milospaunovic/pen/rNGoqWB?editors=1011)

**Reply by mecjos** [reply]
This is the method I use currently.. I just wonder How to do it with table filter?

**Comment by metalsadman** [comment]
https://codepen.io/metalsadman/pen/xxXMRbK?editors=1010
